### ISSUE TITLE
Will not exit with Quit command ("q" or "qq") from command-line "-c" args

### ISSUE THREAD

**Issue by MV10** [microsoftfeedback/WinDbg-Feedback#100]
We have some Powershell scripts that automate the older windbg.exe with a series of commands passed from the command-line using the `-c` switch. A made-up example might be:

```
-z c:\dumps\fubar.dmp -c ".load c:\foo\netext.dll;!wmodule -saveto c:\dumps\modules;q"
                                                         note the "q" quit command ^^
```

However, if these commands are passed to the new DbgX.Shell.exe, the quit command at the end doesn't work. 

Instead, it shows this:

```
Exit a remote ntsd/cdb client with Ctrl-b<enter>.
Exit a remote windbg client with File.Exit (Alt-F4).
If you really want the server to quit use an extra q, such as 'qq' or 'qqd'.

0:000>
```

Well, I _really_ do want it to exit, but changing `q` to `qq` or `qqd` doesn't work, either. Using either of those shows the following instead, and it still does not exit. (I can manually close it, of course, but that defeats the purpose since most of these are unattended scripts, or batches that run over large groups of files, etc.)

```
quit:

?:???> Debuggee not connected
```

I'm still on the October 2020 release (1.0.2001.02001) due to the difficulties of obtaining newer versions in a locked-down corporate environment (aka "breaking corporate security rules and possibly getting fired").

**Comment by lzybkr** [comment]
Updating your debugger won't help, the behavior hasn't changed.

Is there a reason you want to use a graphical debugger? I think most people use `cdb` for your scenario.

**Comment by MV10** [comment]
That make sense, we weren't aware of cdb ... for almost 20 years we've only used windbg, and I was concerned as we've assumed the newer one would eventually replace the older one. Still seems like a regression to me, but for my specific needs I can indeed use cdb. Thanks.

**Comment by lzybkr** [comment]
The change has more to do with how the new WinDbg always runs the debugger engine in a different process.

In the new or old WinDbg, if you were debugging via a remote, `q` would report exactly what are seeing.

The key difference between new and old is that the new WinDbg is always debugging via a (local) remote.

This has many benefits, e.g. you don't have to worry about launching the correct debugger (32 bit vs. 64 bit) and it's easy to start an unrelated session w/o exiting the debugger, but it seems there are some minor issues with this approach as well.

The bottom line though - cdb is better suited for automation which seems to be your scenario, so we probably won't make any changes here.
